scripts/meson-kernel-x86_64-managarm.cross-file

```diff
@@ -5,17 +5,17 @@ ar = 'x86_64-managarm-ar'
 strip = 'x86_64-managarm-strip'
 pkgconfig = 'pkg-config'
 
+[constants]
+# common args that c and cxx need
+args = ['-mno-80387', '-mno-mmx', '-mno-sse', '-mno-sse2', '-ffreestanding']
+# cxx specific arguments
+cxx = args + [ '-target', 'x86_64-managarm-kernel', '-gcc-toolchain', '_BUILD_ROOT_/tools/kernel-gcc']
```

Comment:
The C compiler also needs these target arguments, so that when compiling on a host with a different architecture than x86_64 the assembler knows the architecture of .S files.